Tetfuza/TetfuzaBackend/TetfuzaBackend.cs

```diff
@@ -175,6 +112,9 @@ public long Run()
                 pieceNum = _rand.Next(0, 7);
                 AfterPiece = new FuzaPiece((FuzaType)pieceNum);
 
+                // Draw Board
+                _screen.GameScreen(this);
```

Comment:
What does GameScreen do? Need more descriptive method name

Reply:
My original idea was to have the display implement methods to draw the menu and the game screens specifically, but upon further examination, it appears that we only really need a DrawBoard() and WriteText(xPos, yPos) to do everything we need.